tests/unit/task_scheduler/file/test_backend.py

```
@@ -219,5 +228,119 @@ def test_execute_function_error(self):
+
+    @unittest.skipIf(sys.platform == "win32" or skip_pysqa_test, "pysqa module patching not supported on Windows or when pysqa is not installed")
+    def test_check_task_output_dead_job_reported_as_error_status(self):
+        # On some backends (e.g. flux, whose "flux jobs -a" keeps listing inactive jobs) a
+        # cancelled/failed job is never removed from the queue listing - unlike slurm's squeue,
+        # which drops the job entirely (status None) once it is gone. Instead pysqa reports it
+        # with its terminal-failure status "error" (see command_pysqa.pysqa_terminate, which
+        # already treats "error" as not alive). That status must also fail the future - see
+        # https://github.com/pyiron/executorlib/issues/1037.
+        cache_directory = os.path.abspath("executorlib_cache")
+        os.makedirs(cache_directory, exist_ok=True)
+        task_key, data_dict = serialize_funct(fn=my_funct, fn_args=[1], fn_kwargs={"b": 2})
```

Ruff (0.16.1)

[warning] Unpacked variable `data_dict` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)